permissions/plugin-list-git-branches-parameter.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "list-git-branches-parameter"
+github: "jenkinsci/list-git-branches-parameter-plugin"
+paths:
+- "org/jenkins-ci/plugins/list-git-branches-parameter"
+- "org/jenkins-ci/tools/list-git-branches-parameter"
```

Comment:
The only reason to have multiple paths here is when you change these over time. For a new plugin, this is not necessary, so just choose one and stick to it.